org-formation/730-cloudwatch-to-S3/_tasks.yaml

```diff
@@ -0,0 +1,31 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+  appName:
+    Type: String
+    Default: 'CloudWatch2S3'
+
+  accountId:
+    Type: String
+    Description: The identifier from the account used as the main account
+    Default: !Ref LogCentralAccount
+
+Cloudwatch2S3-main:
+  Type: update-stacks
+  Template: CloudWatch2S3.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}'
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: !Ref accountId
+  Parameters:
```

Comment:
would probably be better to create a bucket before hand then just pass the bucket ARN into the template with `BucketName` parameter.  Creating the bucket would allow us to setup a lifecycle policy and bucket encryption.

Reply:
done